### PR TITLE
Added maps to Reach College page with directions feature.

### PR DESCRIPTION
**Pull request by yasht01**

### CODE REVIEW

**Comment by yasht01** [comment]
Please **dont** merge this PR now. There is one commit labelled "Merge branch 'master' of https://github.com/deprov447/cse-fresher-web". I need to figure this out.

File: frontEnd/public/roadmap.ejs [hunk 1]
```diff
@@ -6,19 +6,63 @@
     <title>Gallery</title>
     <link href="https://fonts.googleapis.com/css2?family=Montserrat:wght@400;600&family=Raleway:wght@400;600&display=swap" rel="stylesheet"> 
 
+    <!-- Mapbox CDN -->
+    <script src='https://api.mapbox.com/mapbox-gl-js/v1.12.0/mapbox-gl.js'></script>
+    <link href='https://api.mapbox.com/mapbox-gl-js/v1.12.0/mapbox-gl.css' rel='stylesheet' />
+
+    <!-- Bootstrap CDN -->
```
Comment (deprov447) [inline frontEnd/public/roadmap.ejs:13]: Do we really need Bootstrap? For responsiveness issue... I've proposed some changes. Have a look

File: frontEnd/public/roadmap.ejs [hunk 2]
```diff
@@ -6,19 +6,63 @@
...
+
   <%- include("partials/nav-up.ejs") %>
   <div class="page-content">
 
     <!-- add page content here -->
     <h1>Map to the college</h1>
-  
-    
     
+    <div id="map" class="col-sm-12 col-lg-6"></div>
```
Comment (deprov447) [inline frontEnd/public/roadmap.ejs]: If bootstrap not needed, remove these classes

File: frontEnd/public/styles/roadmap.css [hunk 3]
```diff
@@ -0,0 +1,8 @@
+#map {
+    margin: 100px auto;
+    position: absolute;
+    top: 0;
+    bottom: 0;
+    width: 500px;
```
Comment (deprov447) [inline frontEnd/public/styles/roadmap.css]: Try Use this instead:

#map {
    width: 100vw;
    height: 100vh;
}

Comment (deprov447) [reply frontEnd/public/styles/roadmap.css]: no need of absolute positioning and margin

Comment (deprov447) [reply frontEnd/public/styles/roadmap.css]: And just for some polishing, try including these too...

body{
    overflow-x: hidden;
}
h1{
    text-align: center;
}

**Review by deprov447 — COMMENTED** [review]
### Maps page after applying requested changes 
maps

File: frontEnd/public/roadmap.ejs (hunk 1 above)
Comment (yasht01) [reply frontEnd/public/roadmap.ejs:13]: Yeah, I was thinking of adding two maps and positioning using Grid system. But, putting two maps increases the API requests two times. So, I dropped that idea but forgot to remove the code

File: frontEnd/public/roadmap.ejs (hunk 2 above)
Comment (yasht01) [reply frontEnd/public/roadmap.ejs]: Ok! I will work one these right now.

**Review by sagar-barapatre — APPROVED** [review]

**Comment by yasht01** [comment]
Hey @deprov447 , I have worked on the suggestions. Have a look at it.

**Review by sagar-barapatre — APPROVED** [review]

**Review by deprov447 — APPROVED** [review]

**Review by Sandeepan-Das — APPROVED** [review]

**Pull request merged**